packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4900,32 +4900,6 @@ export default class Meeting extends StatelessWebexPlugin {
             )
           )
           .then(() => this.checkForStopShare(mediaSettings.sendShare, previousSendShareStatus))
-          .then((startShare) => {
-            // This is a special case if we do an /floor grant followed by /media
-            // we actually get a OFFER from the server and a GLAR condition happens
-            if (startShare) {
-              // We are assuming that the clients are connected when doing an update
-              // @ts-ignore
-              return this.share();
-            }
-
-            return Promise.resolve();
-          })
```

Comment:
was this bit removed on purpose too?
i see that the second .then is a duplicate.

Reply:
we changed the name of the function which cause a merge conflict , accidentally we kept both the code causing the issue , so basically they are the same